src/benchmarks/micro/libraries/Microsoft.Extensions.Configuration/ConfigurationBinderBenchmarks.cs

```diff
@@ -0,0 +1,75 @@
+﻿using BenchmarkDotNet.Attributes;
```

Comment:
This needs a copyright header. See other .cs files in the repo to copy from.

Reply:
Added. Thanks for reminding.